lib/collection/src/common/sha_256.rs

```diff
@@ -25,6 +41,52 @@ pub async fn hash_file(file_path: &Path) -> io::Result<String> {
     Ok(format!("{hash:x}"))
 }
 
+/// Compute sha256 hash for the given file using the `sha256sum` binary
+///
+/// May fail if the binary does not exist on the system.
+#[cfg(all(debug_assertions, unix))]
+async fn hash_file_bin(file_path: &Path) -> io::Result<String> {
+    use tokio::process::Command;
+
+    let output = match Command::new("sha256sum")
+        .arg("-b")
+        .arg(file_path.as_os_str())
```

Comment:
I think just `file_path` would work here.

```suggestion
        .arg(file_path)
```